lib/explorer/series.ex

```diff
@@ -4332,9 +4332,27 @@ defmodule Explorer.Series do
   @doc type: :string_wise
   @spec trim_leading(Series.t()) :: Series.t()
   def trim_leading(%Series{dtype: :string} = series),
-    do: apply_series(series, :trim_leading)
+    do: apply_series(series, :trim_leading, [nil])
 
-  def trim_leading(%Series{dtype: dtype}), do: dtype_error("trim_leading/1", dtype, [:string])
+  @doc """
+  Returns a string where all leading examples of the provided binary have been removed.
```

Comment:
not really sure on the wording of this

Reply:
Maybe something like:

```suggestion
  Returns a series where all leading examples of the provided string have been removed.
```